SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesModule.h

```diff
@@ -24,7 +24,7 @@ class SPATIALGDKSERVICES_API FSpatialGDKServicesModule : public IModuleInterface
 	static FString GetSpatialGDKPluginDirectory(const FString& AppendPath = TEXT(""));
 	static const FString& GetSpotExe();
 	static const FString& GetSpatialExe();
-	static bool SpatialPreRunChecks();
+	static bool SpatialPreRunChecks(bool bIsInChina);
```

Comment:
If there is any other way of doing this I would rather we do that.

Reply:
How about reading the ini value rather than passing a bool around?

Reply:
We would rather open and read a file than pass a bool around?

Reply:
Using <GetDefault>USpatialGDKSettings() will use the bool stored in the class state, these are all read at startup anyway so it's just reading a centralised bool instead of passing it around.

Reply:
Oh right, I will need to check if SpatialGDKServicesModule.h can access USpatialGDKSettings. I don't think it can but maybe.

Reply:
There is a `USpatialGDKServicesSettings` now which I believe we should be using for this actually :D

Reply:
Oh daym. Sounds hot.